scripts/helpers/deps.js

```diff
@@ -43,8 +43,16 @@ module.exports.getNPMDependencies = ({ preserve = false } = {}) => {
 	const isDirectory = source => lstatSync(source).isDirectory();
 	const getDirectories = source => readdirSync(source).map(name => join(source, name)).filter(isDirectory);
 
-	const packages = getDirectories(resolve(process.cwd(), 'packages')).map(path => `${path}/lib/`);
-
+	const packages = getDirectories(resolve(process.cwd(), 'packages')).map(path => {
```

Comment:
Am I right that this code 'fixes' the wrong scaffold provided by our schematics? (part 2)

Reply:
Yep